internal/server/models/destination.go

```diff
@@ -12,6 +12,9 @@ type Destination struct {
 
 	ConnectionURL string
 	ConnectionCA  string
+
+	Resources CommaSeparatedStrings
+	Roles     CommaSeparatedStrings
```

Comment:
I think @dnephin had a comment that was resolved around this, but why not use the native `json` type to store the slice? Less code for us to maintain and we may be able to query those fields now.